src/scheduler/scheduler.service.spec.ts

```
@@ -0,0 +1,209 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { SchedulerService } from './scheduler.service';
+import { InterviewSlot } from './entities/interview-slot.entity';
+import { TimezoneHelper } from './utils/timezone.helper';
+import { IcsGenerator } from './calendar/ics.generator';
+import { ConflictException, NotFoundException } from '@nestjs/common';
+
+describe('SchedulerService', () => {
+  let service: SchedulerService;
+  let slotRepository: Repository<InterviewSlot>;
```

ESLint

[error] 'slotRepository' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)